src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -58,14 +59,14 @@ public void keyboardShortcutsTest() {
         // - To ensure shortcut works from filter box, too
         press(JUMP_TO_FIRST_ISSUE);
         press(JUMP_TO_FILTER_BOX);
-        assertEquals(10, selectedIssueId);
+        assertEquals(DummyRepoState.noOfDummyIssues, selectedIssueId);
         clearSelectedIssueId();
 
         // jump to first issue using number key(1) or ENTER
         push(KeyCode.ESCAPE);
         press(JUMP_TO_NTH_ISSUE_KEYS.get(1));
         PlatformEx.waitOnFxThread();
-        assertEquals(10, selectedIssueId);
```

Comment:
This entire block of `JUMP_TO_NTH_ISSUE_KEYS` can be put into a while-loop. Much cleaner!